components/Navigation.tsx

```diff
@@ -116,7 +116,9 @@ function NavigationSubLink({ href, children }: NavigationSubLinkProps) {
     <span className="flex items-center ml-4">
       <Link href={href}>
         <a className={classes}>
-          {router.asPath === href ? <Caret className="mr-1 w-2 h-2" /> : null}
+          {router.asPath === href ? (
+            <Caret className="ml--1 absolute -m-4 mr-1 w-2 h-2" />
```

Comment:
think the `ml--1` can be removed, doesn't do anything (should it be `-ml-4`?

Reply:
lol, I can't believe I left that in. Fixed that and added the relative to the parent so the absolute is in the right place 😄

Reply:
Just delete the ml--1 and then merge this sucker!

Reply:
how in the WHAT is it still there

Reply:
oh it isn't